testsuites/CBLTester/System_test_multiple_device/test_system_testing.py

```diff
@@ -0,0 +1,283 @@
+                x = ["{}_{}".format(doc_prefix, doc_id) for doc_id in range((j + 1) * num_of_docs_in_itr, ((j + 1) * num_of_docs_in_itr) + extra_docs_in_itr_per_db)]
+                assert sorted(ids) == sorted(x)
+                doc_ids.update(ids)
+        # add the extra docs to last db
+        if extra_docs != 0:
+            ids = db_obj.create_bulk_docs(extra_docs, "cbl_{}".format(db_name), db=cbl_db, channels=channels_sg, id_start_num=docs_per_db, generator=generator)
+            x = ["{}_{}".format(doc_prefix, doc_id) for doc_id in range(docs_per_db, docs_per_db + extra_docs_in_itr_per_db)]
+            assert sorted(ids) == sorted(x)
+            doc_ids.update(ids)
+    else:
+        # getting doc ids from the dbs
+        _check_doc_count(db_obj_list, cbl_db_list)
+        count = db_obj_list[0].getCount(cbl_db_list[0])
+        itr_count = count / query_limit
+        if itr_count == 0:
+            itr_count = 1
+        for _ in range(itr_count):
+            existing_docs = db_obj_list[0].getDocIds(cbl_db_list[0], query_limit, query_offset)
+            doc_ids.update(existing_docs)
+            query_offset += query_limit
+        print "{} Docs in DB".format(len(doc_ids))
+        query_offset = 0
+        try:
+            # Precautionary creation of user
+            sg_client.create_user(sg_admin_url, sg_db, username, password, channels=channels_sg)
+        except:
+            pass
+    # Configure replication with push_pull for all db
+    replicator_obj_list = []
+    replicator_list = []
+    for base_url, cbl_db, query in zip(base_url_list, cbl_db_list, query_obj_list):
+        repl_obj = Replication(base_url)
+        replicator_obj_list.append(repl_obj)
+        session, _, repl = repl_obj.create_session_configure_replicate(
+            base_url, sg_admin_url, sg_db, username, password, channels_sg, sg_client, cbl_db, sg_blip_url, continuous=True)
+        replicator_list.append(repl)
+        query.query_get_docs_limit_offset(cbl_db, limit=query_limit, offset=query_offset)
+
+    current_time = datetime.now()
+    running_time = current_time + timedelta(minutes=up_time)
+
+    _check_doc_count(db_obj_list, cbl_db_list)
+    x = 1
+    while(running_time - current_time > timedelta(0)):
+
+        print '*' * 20
+        print "Starting iteration no. {} of system testing".format(x)
+        print '*' * 20
+        x += 1
+        ######################################
+        # Checking for doc update on SG side #
+        ######################################
+        docs_to_update = random.sample(doc_ids, num_of_docs_to_update)
+        sg_docs = sg_client.get_all_docs(url=sg_url, db=sg_db, auth=session)["rows"]
+        sg_docs = [doc for doc in sg_docs if doc["id"] in docs_to_update]
+        print "Updating {} docs on SG - {}".format(len(docs_to_update),
```

Comment:
can you either remove print statements or replace with log_info?

Reply:
changed it to log_info